src/services/getVisa.ts

```diff
@@ -0,0 +1,61 @@
+import { isNil, omitBy } from 'lodash';
+import queryString from 'querystring';
+
+import ajax from 'services/ajax';
+
+import { VisaPermission } from 'common/typedefs/VisaPermission';
+import { clientSideSort } from './clientSideSortUtil';
+
+export const getVisa = (id) => {
+  return ajax
+    .get(`/visas/${id}`)
+    .then((r) => r.data)
+    .catch((err) => err);
+};
+
+export const getVisaPermissions = ({
```

Comment:
We have a system here for one request per file, this file has two requests. Should we move this to another file?